### DISCUSSION TITLE
Is a Win2 Version Obsolete?

### DISCUSSION THREAD

**Discussion by DavidDragonsage** [metercai/SimpleSDXL#84] [Q&A]
SimpleSDXL2 0820 had a "win" version and a "win2" version. "win2" was necessary as a fallback version for those who had problems running the win version, but some of its images had less detail, less clarity and reduced prompt compliance.

Currently SimpleSDXL2 0916 only has one version. When running SDXL models, the results are identical to the 0820 "win" version, which is great! Will a fallback version of 0916 be necessary?

So far no one at the Pure Fooocus Facebook group has complained about not being able to run 0916.

**Comment by metercai** [comment]
> SimpleSDXL2 0820 had a "win" version and a "win2" version. "win2" was necessary as a fallback version for those who had problems running the win version, but some of its images had less detail, less clarity and reduced prompt compliance.
> 
> Currently SimpleSDXL2 0906 only has one version. When running SDXL models, the results are identical to the 0820 "win" version, which is great! Will a fallback version of 0906 be necessary?
Unnecessary. There will be a Pytorch 2.4.1+cu124 package in the future.

> 
> So far no one at the Pure Fooocus Facebook group has complained about not being able to run 0906.
Ok, correct it, it's 0916

**Comment by DavidDragonsage** [comment]
😁 And the answer is?

**Comment by DavidDragonsage** [comment]
SimpleSDXL2_0820 is no longer available in either version so I guess I finally received an answer. 😏
